components/form.tsx

```diff
@@ -7,11 +7,18 @@ import styles from './form.module.css';
 import { postTestimonial } from '../services/airTableService';
 import testimonial from '../interfaces/testimonial';
 
-const FormContainer = styled.div`
+const FormContainer = styled.form`
   margin: 2rem 2rem;
   width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
 `;
 
+const Headings = styled.form``;
```

Comment:
this is confusing if you're going to use this, a component called Headings thats actually a `form` element